packages/shared/config.ts

```diff
@@ -56,6 +56,12 @@ const allEnv = z.object({
   DATA_DIR: z.string().default(""),
   MAX_ASSET_SIZE_MB: z.coerce.number().default(4),
   INFERENCE_LANG: z.string().default("english"),
+  WEBHOOK_URLS: z.preprocess(
+    (val) => typeof val === "string" ? val.split(",") : val,
+    z.array(z.string().url())
```

Comment:
Hmmm, not sure if the use of `preprocess` here is right. It seems more correct to do do:

```
z.string().transform(val => val.split(",")).pipe(z.array(z.string().url()))
```